docs/user-guide/connecting-applications.md

```diff
@@ -393,6 +393,165 @@ LoadBalancer Ingress:   a320587ffd19711e5a37606cf4a74574-1142138393.us-east-1.el
 ...
 ```
 
+## Loadbalancing multiple Services
+
+Till now we had restricted ourselves to a single Service, but Kubernetes has a resource called [Ingress](/docs/user-guide/ingress) that spans Services and loadbalances requests at [layer 7](https://en.wikipedia.org/wiki/OSI_model#Layer_7:_Application_Layer). This is useful for reasons like:
```

Comment:
Just start with:

Kubernetes has a resource called [Ingress](/docs/user-guide/ingress) that spans Services and load-balances requests at [layer 7](https://en.wikipedia.org/wiki/OSI_model#Layer_7:_Application_Layer).